manifests/testing-framework/testing-framework.yaml

```diff
@@ -84,6 +84,12 @@ spec:
         - name: rolling-update-2
           secret:
             secretName: rolling-update-2
+        - name: proxy-with-hetzner
+          secret:
+            secretName: proxy-with-hetzner
+        - name: proxy-without-hetzner
+          secret:
+            secretName: proxy-without-hetzner
```

Comment:
You also need to reference this in the `volumeMounts` in https://github.com/berops/claudie/blob/741945a1eb77c8355785e5715339e9b666c835bb/manifests/testing-framework/testing-framework.yaml